scripts/mirror.js

```diff
@@ -144,9 +144,7 @@ const combineNotes = (notes1, notes2) => {
     }
   }
 
-  newNotes = newNotes.filter((item, pos) => {
-    newNotes.indexOf(item) == pos;
-  });
+  newNotes = newNotes.filter((item, pos) => newNotes.indexOf(item) == pos);
```

Comment:
Does this deduplicate the notes? And wasn't the old implementation broken, i.e. the block returned `undefined` for every note?

Reply:
That's correct, for both!